synfig-studio/src/gui/dialogs/canvasresize.cpp

```diff
@@ -166,7 +167,19 @@ void CanvasResize::on_dialog_shown()
 {
 	if (is_image_checked) set_image_flags(toggle_ratio_wh->get_active());
 
-	// Always enable width/height link
+	/*
```

Comment:
1) I suggest moving this comment next to the variable declaration so it can be found from anywhere.
2) Maybe name this variable something like `switch_changed_manually`? This should better describe it.
3) It looks like this code could be simplified somehow, but I need to test some ideas. So I will back tomorrow.

Reply:
@Keyikedalube ping =]

Reply:
Waiting for improvement highlighted in point 3 LOL

Reply:
Yikes! I missed step 2...
Step 1 done